.gitignore

```diff
@@ -2,7 +2,7 @@
 src/_css/img/safari.png
 src/_css/img/chrome.svg
 src/_css/img/firefox.svg
-src/_js/dark-mode-toggle.mjs
+src/_js/dark-mode-toggle.min.mjs
```

Comment:
?

Reply:
This is actually no longer required. Removing and resolving the conversation…